error TS2440: Import declaration conflicts with local declaration of 'AccessScope'

<!--
BUGS: Please use this template.

SUPPORT REQUESTS/QUESTIONS: If you have a support request or question please
submit them to StackOverflow using the tags aurelia and aurelia-framework
http://stackoverflow.com/questions/tagged/aurelia
or the Aurelia Gitter https://gitter.im/aurelia/discuss

Blue Spire offers paid support agreements. Further information regarding paid support
may be obtained by emailing support@bluespire.com

Future support requests submitted here will be closed.

The HTML comments below are for your reference, and are not displayed
when your issue is submitted, feel free to leave them.

Choose one of the two headings, delete the other.
-->
**I'm submitting a bug report**

* **Library Version:**
1.5.0


**Please tell us about your environment:**
* **Operating System:**
Windows 10

* **Node Version:**
10.8.0
<!--
Minimum supported Node version is latest Node 4.x LTS
run `node -v`
-->

* **NPM Version:**
6.2.0
<!--
Minimum supported NPM version is 3.x
run `npm -v`
-->

* **JSPM OR Webpack AND Version**
Actually still using gulp v4.0.0 from old way to setup aurelia-cli build
<!--
If using JSPM
run `jspm -v`
If using Webpack
run `webpack --help | grep webpack`
-->

* **Language:**
TypeScript 3.7

**Current behavior:**
Since upgrading to TypeScript 3.7, I am getting the following error during transpilation:
`node_modules/aurelia-validation/dist/aurelia-validation.d.ts(1,37): error TS2440: Import declaration conflicts with local declaration of 'AccessScope'`.

If I comment out https://github.com/aurelia/validation/blob/master/dist/aurelia-validation.d.ts#L505 the transpilation works (though obviously that's not an actual solution).

**Expected/desired behavior:**
TypeScript 3.7 transpilation to work.
<!--
If the current behavior is a bug, please provide the steps to reproduce and, if possible, a minimal demo of the
problem using a runnable Gist.
To create a runnable gist, go to https://gist.run/?id=381fdb1a4b0865a4c25026187db865ce
Then click "Fork to Public Gist".
Create your gist, then finally click "Update Gist." Include a link to the gist.run below.
-->

**tsconfig.json**
```
{
  "compileOnSave": false,
  "compilerOptions": {
    "sourceMap": true,
    "target": "es5",
    "module": "amd",
    "declaration": false,
    "noImplicitAny": false,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "downlevelIteration": true,
    "moduleResolution": "node",
    "lib": ["es2019", "dom"],
    "baseUrl": "./src",
    "paths": {
      "@azure/storage-blob": ["../lib/azure-storage.blob.min.js"],
      "*": ["*", "../*"]
    }
  },
  "exclude": [
    "node_modules",
    "aurelia_project"
  ],
  "filesGlob": [
    "./src/**/*.ts",
    "./test/**/*.ts",
    "./custom_typings/**/*.d.ts"
  ],
  "atom": {
    "rewriteTsconfig": false
  }
}
```

**package.json**
```
{
  "name": "web",
  "description": "An Aurelia client application.",
  "version": "0.1.0",
  "repository": {
    "type": "???",
    "url": "???"
  },
  "license": "MIT",
  "dependencies": {
    "@aspnet/signalr": "^1.1.4",
    "@types/bootstrap": "^4.3.0",
    "@types/daterangepicker": "^3.0.1",
    "applicationinsights-js": "1.0.8",
    "aurelia-animator-css": "^1.0.1",
    "aurelia-bootstrapper": "^2.3.1",
    "aurelia-dialog": "^2.0.0-rc.3",
    "aurelia-fetch-client": "^1.7.0",
    "aurelia-store": "^1.3.1",
    "aurelia-validation": "^1.5.0",
    "bluebird": "^3.5.3",
    "bootstrap": "^4.3.1",
    "bootstrap-select": "^1.13.10",
    "daterangepicker": "^3.0.3",
    "highcharts": "^7.0.0",
    "jquery": "^3.3.1",
    "moment": "^2.22.1",
    "popper.js": "^1.14.6",
    "requirejs": "^2.3.6",
    "text": "github:requirejs/text#latest",
    "ts-keycode-enum": "^1.0.6"
  },
  "peerDependencies": {},
  "devDependencies": {
    "@types/applicationinsights-js": "^1.0.9",
    "@types/bootstrap-select": "^1.11.1",
    "@types/jest": "^23.3.10",
    "@types/node": "^12.12.6",
    "aurelia-cli": "^1.0.0-beta.7",
    "aurelia-pal-nodejs": "^1.2.0",
    "aurelia-testing": "^1.0.0",
    "aurelia-tools": "^2.0.0",
    "browser-sync": "^2.26.3",
    "connect-history-api-fallback": "^1.2.0",
    "event-stream": "^4.0.1",
    "gulp": "github:gulpjs/gulp#v4.0.0",
    "gulp-changed-in-place": "^2.0.3",
    "gulp-notify": "^3.2.0",
    "gulp-plumber": "^1.2.1",
    "gulp-rename": "^1.2.2",
    "gulp-replace": "^1.0.0",
    "gulp-sass": "^4.0.2",
    "gulp-sourcemaps": "^2.0.0-alpha",
    "gulp-tslint": "^8.1.3",
    "gulp-typescript": "^5.0.0",
    "jest": "^23.6.0",
    "jest-cli": "^23.6.0",
    "jest-teamcity-reporter": "^0.9.0",
    "minimatch": "^3.0.2",
    "node-fetch": "^2.3.0",
    "through2": "^3.0.0",
    "ts-jest": "^23.10.5",
    "tslint": "^5.11.0",
    "typescript": "^3.7.2",
    "uglify-js": "^3.4.9",
    "vinyl-fs": "^3.0.3"
  },
  "scripts": {
    "test": "jest --verbose"
  }
}
```

Thanks!

--- a/src/implementation/expression-visitor.ts
+++ b/src/implementation/expression-visitor.ts
@@ -1,40 +1,20 @@
 import {
   Expression,
-  // Chain,
   ValueConverter,
-  // Assign,
   Conditional,
-  // AccessThis,
-  // AccessScope,
+  AccessScope,
   AccessMember,
   AccessKeyed,
-  // CallScope,
-  // CallFunction,
   CallMember,
-  // PrefixNot,
   BindingBehavior,
   Binary,
-  // LiteralPrimitive,
-  // LiteralArray,
-  // LiteralObject,
-  // LiteralString
+  LiteralPrimitive,
+  LiteralString
 } from 'aurelia-binding';
-
-export type Chain = any;
-export type Assign = any;
-export type AccessThis = any;
-export type AccessScope = any;
-export type CallScope = any;
-export type CallFunction = any;
-export type PrefixNot = any;
-export type LiteralPrimitive = any;
-export type LiteralArray = any;
-export type LiteralObject = any;
-export type LiteralString = any;
 
 // tslint:disable:no-empty
 export class ExpressionVisitor {
-  public visitChain(chain: Chain) {
+  public visitChain(chain: any) {
     this.visitArgs(chain.expressions);
   }
 
@@ -48,7 +28,7 @@ export class ExpressionVisitor {
     this.visitArgs(converter.args);
   }
 
-  public visitAssign(assign: Assign) {
+  public visitAssign(assign: any) {
     assign.target.accept(this);
     assign.value.accept(this);
   }
@@ -59,7 +39,7 @@ export class ExpressionVisitor {
     conditional.no.accept(this);
   }
 
-  public visitAccessThis(access: AccessThis) {
+  public visitAccessThis(access: any) {
     access.ancestor = access.ancestor;
   }
 
@@ -76,11 +56,11 @@ export class ExpressionVisitor {
     access.key.accept(this);
   }
 
-  public visitCallScope(call: CallScope) {
+  public visitCallScope(call: any) {
     this.visitArgs(call.args);
   }
 
-  public visitCallFunction(call: CallFunction) {
+  public visitCallFunction(call: any) {
     call.func.accept(this);
     this.visitArgs(call.args);
   }
@@ -90,7 +70,7 @@ export class ExpressionVisitor {
     this.visitArgs(call.args);
   }
 
-  public visitPrefix(prefix: PrefixNot) {
+  public visitPrefix(prefix: any) {
     prefix.expression.accept(this);
   }
 
@@ -103,11 +83,11 @@ export class ExpressionVisitor {
     literal.value = literal.value;
   }
 
-  public visitLiteralArray(literal: LiteralArray) {
+  public visitLiteralArray(literal: any) {
     this.visitArgs(literal.elements);
   }
 
-  public visitLiteralObject(literal: LiteralObject) {
+  public visitLiteralObject(literal: any) {
     this.visitArgs(literal.values);
   }
 

